src/main/templates/assets/projecty/models/block/icosahedron/model_1.mtl

```diff
@@ -0,0 +1,8 @@
+model_0.mtl# Blender MTL File: 'None'
```

Comment:
It seems like this would cause a model loading error?

Reply:
fixed in dev